Dockerfile

```diff
@@ -21,6 +21,17 @@ RUN ln -s /opt/python/cp${PYV}* /opt/python/v
 RUN pip install future numpy setuptools wheel && \
     rm -rf /root/.cache/pip/
 
+RUN if [ ${PYV} != "37" ] ; then \
```

Comment:
1.13 and 1.14 works for Python 3.7 so it need a bit of rework.

Reply:
ok, for now it's only for testing